### PR TITLE
BUG: Validate the justify parameter in to_html

### LINKED ISSUE #17527: DOC: "justify" parameter in to_html
The Travis docs for <a href="https://pandas-docs.github.io/pandas-docs-travis/generated/pandas.DataFrame.to_html.html">to_html</a> state that the only accepted parameters for `justify` are `left` and `right`.

However, I stumbled upon the fact that you can also pass in `center`, and it will work just fine (i.e. the cell elements and headers are centered), as evidenced by the code:

https://github.com/pandas-dev/pandas/blob/fa557f7391589f351b1260f46b3b3db22492f50b/pandas/io/formats/format.py#L1215-L1217

@TomAugspurger : Should we continue to allow only `left` and `right` or should we expand to accept `center` ?  I'm personally up for the latter but wanted to get consensus first.

### PR DESCRIPTION
**Pull request by gfyoung**
Closes #17527

cc @TomAugspurger

### CODE REVIEW

File: doc/source/whatsnew/v0.21.0.txt [hunk 1]
```diff
@@ -778,6 +778,7 @@ I/O
 - Bug in :func:`read_html` where import check fails when run in multiple threads (:issue:`16928`)
 - Bug in :func:`read_csv` where automatic delimiter detection caused a ``TypeError`` to be thrown when a bad line was encountered rather than the correct error message (:issue:`13374`)
 - Bug in ``DataFrame.to_html()`` with ``notebook=True`` where DataFrames with named indices or non-MultiIndex indices had undesired horizontal or vertical alignment for column or row labels, respectively (:issue:`16792`)
+- Bug in ``DataFrame.to_html()`` in which there was no validation of the ``justify`` parameter (:issue:`17527`)
```
Comment (jschendel) [inline doc/source/whatsnew/v0.21.0.txt]: Would it be preferable to use <code>:meth:\`DataFrame.to_html\`</code> to provide a link?  Possibly on the line above too?

Comment (gfyoung) [reply doc/source/whatsnew/v0.21.0.txt]: Absolutely!  Thanks for pointing that out.

**Comment by jreback** [comment]
lgtm. merge on green.

**Comment by gfyoung** [comment]
@jreback , @TomAugspurger : All green, so merging.

**Pull request merged**